test.py

```diff
@@ -0,0 +1,69 @@
+import re
+import unittest
```

Comment:
for e2e testing recommend using 'pytest' or 'behave' library

Reply:
Should I switch to one of these libraries?